test/common/upstream/host_stats_test.cc

```diff
@@ -9,30 +9,25 @@ namespace {
 // Verify that counters are sorted by name.
 TEST(HostStatsTest, CountersSortedByName) {
   HostStats host_stats;
-  auto counters = host_stats.counters();
+  using PrimitiveCounterReference = std::reference_wrapper<const Stats::PrimitiveCounter>;
```

Comment:
I meant putting this nickname in the API for your new class, not at the call-site :)

Reply:
Moved to the new class.  If you want additional aliases added, please specify exactly which ones.

Reply:
Thanks all set now :)